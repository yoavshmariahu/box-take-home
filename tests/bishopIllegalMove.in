k a1
K e5
b d1
B b5

[G S R P]
[p r s g]

move d1 d2
