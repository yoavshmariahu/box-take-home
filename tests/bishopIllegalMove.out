lower player action: move d1 d2
5 |__| B|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k|__|__| b|__|
    a  b  c  d  e

Captures UPPER: G S R P
Captures lower: p r s g

UPPER player wins.  Illegal move.
